proto/decentraland/social_service_v2/social_service.proto

```diff
@@ -88,6 +113,22 @@ message FriendshipUpdate {
   }
 }
 
+message GetFriendshipStatusPayload {
+  User user = 1;
```

Comment:
We must consider that, in the future, we will not only request the status of a single user but a list of all our blocked users at app start, so we can block the avatars in-world. It is OK to keep the request of a single user tho, as we will use it whenever we open the context menu of a specific user.

Reply:
Sounds good!